dashboard/src/views/Projects/Versions/index.tsx

```diff
@@ -0,0 +1,172 @@
+          py: 0.5,
+          display: { xs: "none", sm: "flex" },
+          flexWrap: "wrap",
+          gap: 1.5,
+          "& > *": {
+            minWidth: { xs: "120px", md: "160px" },
+          },
+          flexDirection: "column",
+        }}
+      >
+        <Typography
+          level="h2"
+          component="h1"
+          sx={{
+            alignSelf: "center",
+            fontSize: "2rem",
+            color: (t) => t.palette.primary[400],
+            m: "0 1.5rem 2rem 0",
+            border: "1px solid",
+            borderColor: (t) => t.palette.primary[400],
+            p: "1rem 2.5rem",
+            borderRadius: "sm",
+          }}
+        >
+          {project?.name ? `${project.name} versions` : "Versions"}
+        </Typography>
+
+        <FormControl sx={{ flex: 1, pb: "1rem" }} size="sm">
+          <FormLabel>Search for version</FormLabel>
+
+          <Input
+            size="sm"
+            placeholder="Look up for version name"
+            startDecorator={<SearchIcon />}
+            onChange={(e) => setSearchTerm(e.target.value)}
+            value={searchTerm}
+          />
+        </FormControl>
+      </Box>
+
+      {projectId && <ManageVersionModal projectId={projectId} />}
+
+      <Sheet
+        variant="outlined"
+        sx={{
+          display: { xs: "initial", sm: "initial" },
+          width: "100%",
+          borderRadius: "sm",
+          flexShrink: 1,
+          overflow: "auto",
+          minHeight: 0,
+        }}
+      >
+        <Table
+          aria-labelledby="tableTitle"
+          stickyHeader
+          hoverRow
+          sx={{
+            "--TableCell-headBackground":
+              "var(--joy-palette-background-level1)",
+            "--Table-headerUnderlineThickness": "1px",
+            "--TableRow-hoverBackground":
+              "var(--joy-palette-background-level1)",
+            "--TableCell-paddingY": "4px",
+            "--TableCell-paddingX": "8px",
+          }}
+        >
+          <thead>
+            <tr>
+              <th style={{ width: 240, padding: "0.7rem 1.5rem" }}>Name</th>
+              <th style={{ width: 340, padding: "0.7rem 0.5rem" }}>
+                Created At
+              </th>
+              <th
+                style={{
+                  width: 140,
+                  padding: "0.7rem 4.5rem",
+                  textAlign: "end",
+                }}
+              ></th>
+            </tr>
+          </thead>
+
+          {projectId && (
+            <tbody>
+              {paginatedVersions.map((version: Version) => (
+                <VersionTableRow
+                  key={version.id}
+                  projectId={projectId}
+                  versionId={version.id}
+                  versionName={version.name}
+                  createdAt={version.createdAt}
+                />
+              ))}
+            </tbody>
+          )}
+        </Table>
+      </Sheet>
+
+      <TablePaginationSection
```

Comment:
Seems like there exists a MUI version you could be using? https://mui.com/material-ui/react-pagination/

Or this one: https://mui.com/material-ui/react-pagination/#table-pagination for tables specifically

Reply:
decided to do it asap now so skipping it but thanks for heads up!